python/nav/ipdevpoll/jobs.py

```diff
@@ -81,7 +81,8 @@ class JobHandler(object):
 
     def __init__(self, name, netbox, plugins=None, interval=None):
         self.name = name
-        self.netbox = netbox
+        self.netbox_id = netbox
```

Comment:
This is not the cause of the problem. The problem is actually caused by NetboxJobScheduler not being on the stack when JobHandler is initialized. Pull request is update with a change that resets the log context in JobHandler